web/templates/message_avatar.hbs

```diff
@@ -1,4 +1,6 @@
-<div class="u-{{msg/sender_id}} sender_info_hover inline_profile_picture {{#sender_is_guest}} guest-avatar{{/sender_is_guest}} view_user_card_tooltip" aria-hidden="true" data-tooltip-template-id="view-user-card-tooltip-template">
-    <img src="{{small_avatar_url}}" alt="" class="no-drag"/>
+<div class="u-{{msg/sender_id}} message-avatar sender_info_hover view_user_card_tooltip" aria-hidden="true" data-tooltip-template-id="view-user-card-tooltip-template">
+    <div class="inline_profile_picture {{#sender_is_guest}} guest-avatar{{/sender_is_guest}}">
+        <img src="{{small_avatar_url}}" alt="" class="no-drag"/>
+    </div>
 </div>
```

Comment:
There's some references in `web/src/popovers.js` to `inline_profile_picture` that I kinda suspect need to be changed to `message-avatar`; can you grep and investigate?